scripts/Muon/GUI/Common/muon_context/muon_context.py

```diff
@@ -49,9 +56,84 @@ def printContext(self):
         print(LoadText, self.common_context[LoadText])
         print(Groups)
         for group in self.common_context[Groups]:
-            print(group.name)
+            group.Print()
```

Comment:
Is there a reason we need to print everything as we save?

Reply:
this is not for saving and is just part of the print context method.

Reply:
The print statements seem to come out in the results log when it saves though.

Reply:
should be fixed now, I had a print statement inside of the save method